etc/scripts/installation-functions.sh

```diff
@@ -136,32 +161,36 @@ function install_olm {
     rm -rf "$OLM_DIR"
     git clone https://github.com/operator-framework/operator-lifecycle-manager "$OLM_DIR"
     pushd $OLM_DIR; git checkout eaf605cca864e; popd
-    for i in "$OLM_DIR"/deploy/okd/manifests/latest/*.crd.yaml; do oc apply -f $i; done
-    for i in $(find "$OLM_DIR/deploy/okd/manifests/latest/" -type f ! -name "*crd.yaml" | sort); do oc create -f $i; done
+    for i in "$OLM_DIR"/deploy/okd/manifests/latest/*.crd.yaml; do $CMD apply -f $i; done
+    for i in $(find "$OLM_DIR/deploy/okd/manifests/latest/" -type f ! -name "*crd.yaml" | sort); do $CMD create -f $i; done
     wait_for_all_pods openshift-operator-lifecycle-manager
     # perms required by the OLM console: $OLM_DIR/scripts/run_console_local.sh
-    oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:kube-system:default
+    # oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:kube-system:default
     # check our namespace
     OLM_NS=$(grep "catalog_namespace:" "$OLM_DIR/deploy/okd/values.yaml" | awk '{print $2}')
   fi
   # and finally apply the catalog sources
-  oc apply -f "$ROOT_DIR/knative-operators.catalogsource.yaml" -n "$OLM_NS"
-  oc apply -f "$ROOT_DIR/maistra-operators.catalogsource.yaml" -n "$OLM_NS"
+  $CMD apply -f "$ROOT_DIR/knative-operators.catalogsource.yaml" -n "$OLM_NS"
+  $CMD apply -f "$ROOT_DIR/maistra-operators.catalogsource.yaml" -n "$OLM_NS"
 }
 
 function install_istio {
-  # istio
-  oc create ns istio-operator
-  if check_operatorgroups; then
-    cat <<-EOF | oc apply -f -
+  if check_minikube; then
+    echo "Detected minikube - incompatible with Maistra operator, so installing upstream istio."
+    $CMD apply -f "https://github.com/knative/serving/releases/download/${KNATIVE_SERVING_VERSION}/istio.yaml"
```

Comment:
@jcrossley3  can we add the crds' install to avoid initial warning/errors that user might see because of the missing crds ? 
`$CMD kubectl apply --filename https://github.com/knative/serving/releases/download/v0.3.0/istio-crds.yaml`

Reply:
Good call!